core/src/main/resources/org/apache/spark/ui/static/spark-dag-viz.css

```diff
@@ -22,6 +22,7 @@
 #dag-viz-graph .label {
   font-weight: normal;
   text-shadow: none;
+  color: #333;
```

Comment:
What does this change?

According to the PR description, `For RDDs, the color is #212529 which is specified by bootstrap.min.js`.

Reply:
Never mind. I validate this overrides correctly via Web Inspector and UI.

Reply:
Actually, the library DAG-viz uses (dagre-d3) has some mode.
One is text which is the default mode and another is html mode.
With html mode, label is represented as HTML, while with text mode, label is represented as SVG.
For each node in DAG-viz for RDDs, outer label is SVG and inner one is HTML.
The color of font for SVG is specified by `fill` while `color` for HTML so we need this setting.
<img width="389" alt="stage-graph-fixed" src="https://user-images.githubusercontent.com/4736016/80324217-aab32700-886a-11ea-911a-95efc84bf34d.png">


While I wrote the explanation above, I noticed the font color of outer node for query plans is not specified. So, the default font color of SVG seems to be applied (Can you recognize the color of `WholeStageCodegen (1)` is a little bit darker then `SerializeFromObject` ?).
<img width="456" alt="plan-graph-fixed" src="https://user-images.githubusercontent.com/4736016/80324164-6d4e9980-886a-11ea-84d1-4c73e53736b2.png">

I'll create a followup PR.